README.md

```diff
@@ -16,24 +16,8 @@ Python module for modelling and analysing the energy system of offshore oil and
 Part of the [Low Emission Centre](https://www.sintef.no/en/projects/lowemission-research-centre/) (SP5).
 
 ## Getting started
-
-Pypi distribution to come. See local installation below.
-
-## Local installation
-Prerequisite: [Poetry](https://python-poetry.org/docs/#installation)
-
-Clone or download the code and install it as a python package. I.e. navigate to the folder with the MANIFEST.in file and type:
-
-### Install as a package for normal use:
-1. `poetry install`
-
-### Install dependencies for local development
-2. `poetry install --no-root`  --no-root to not install the package itself, only the dependencies.
-3. `poetry shell`
-4. `poetry run pytests tests`
-
-### Local development in Docker
-Alternatively you can run and develop the code using docker and the Dockerfile in the root folder.
+Install latest Oogeso release from PyPi:
+`pip install oogeso`
 
 ## User guide
```

Comment:
The relative links below don't work on pypi, so should use absolute urls for user guide, manual and examples, e.g. `https://github.com/oogeso/oogeso/blob/master/userguide.md` instead of just `userguide.md`

Reply:
Updated.